module/VuFindSearch/src/VuFindSearch/Command/CallMethodCommand.php

```diff
@@ -58,6 +59,13 @@ abstract class CallMethodCommand extends AbstractBase
      */
     protected $method;
 
+    /**
+     * Optional search details.
+     *
+     * @var null|array
```

Comment:
Would `?array` be more concise than `null|array` here and in the associated function?

Reply:
Done